docs/second_factor_auth.md

```diff
@@ -0,0 +1,54 @@
+# Second Factor Authentication (2FA)
+
+According with [wikipedia](https://en.wikipedia.org/wiki/Multi-factor_authentication) Two-factor authentication (also known as 2FA) is a type (subset) of multi-factor authentication. It is a method of confirming a user's claimed identity by utilizing a combination of two different factors: 1) something they know, 2) something they have, or 3) something they are.
+
+This feature provider 2FA for OpenStack Keystone based on email.
+
+## Configuration
+
+Specific options at /etc/keystone/keystone.conf
+```
+[spassword]
+sndfa=true
+sndfa_time_window=24
+sndfa_endpoint='localhost:5001'
+```
+
+`sndfa` is a boolean which enables (true) or disables (false) if Second Factor Authentication feature is available in Keystone instance.
+`sndfa_time_window` indicates the time in hours in which Second Factor Authentication performed by an user is still valid before ask another new one.
+`sndfa_endpoint` is the endpoint used in links sent by email to users to check email address and sndfa changes
+
+
+In order to work with Second Factor Authentication feature spassword needs a proper smtp configuration; make sure that you provide one.
+
+```
+[spassword]
+smtp_server=localhost
+smtp_port=587
+smtp_tls=true
+smtp_user='smtpuser@yourdomain.com'
+smtp_password='yourpassword'
+smtp_from='smtpuser'
+```
+
+## API
```

Comment:
It seems there are some problems rendering this section:

![imagen](https://user-images.githubusercontent.com/1534240/37779413-9d086d8e-2dec-11e8-85d9-21fa4565471f.png)

Not sure if  `<this>` is correctly rendered in github... you can use code-style (i.e. using ```) or use `{alternative}`.

Reply:
fixed in https://github.com/telefonicaid/fiware-keystone-spassword/pull/57/commits/7eeef74282622cfb58b1b66d7a24eca8a7390359